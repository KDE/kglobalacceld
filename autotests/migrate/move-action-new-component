[kstuff]
Launch=Meta+K,Meta+K,Launch

[org.kde.foo.desktop]
_launch=none,Meta+P,Foo
Toggle=Meta+Y,none,Toggle
