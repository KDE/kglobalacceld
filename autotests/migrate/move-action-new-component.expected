[kstuff]
Launch=Meta+K,Meta+K,Launch

[org.kde.foo.desktop]
_launch=none,Meta+P,Foo

[org.kde.bar.desktop]
Toggle=Meta+Y,none,Toggle
